### PR TITLE
footer changes and deletion of conflict code, resolves #11

### LINKED ISSUE #11: footer is floating
http://screencast.com/t/hzqiHpnEmQ

### PR DESCRIPTION
**Pull request by sugarPhlox**
Fix floaty footer issue, and deleted some code from conflict issues.

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -238,3 +221,12 @@ header {
 .affix-top {
   position: static;
 }
+
+.footer {
+  position: absolute;
```
Comment (mowchan) [inline css/main.css]: Why are you using absolute positioning here?

File: css/main.css [hunk 2]
```diff
@@ -7,6 +12,7 @@ body {
     -o-background-size: cover;
     background-size: cover;
   font-family: 'Quattrocento', sans-serif;
+  margin-bottom: 0 0 25px;
```
Comment (mowchan) [inline css/main.css]: This should just be 'margin-bottom: 25px' or 'margin: 0 0 25px'. 

You should also consider increasing this number to something like 75px so there is more space between the content and footer.

File: css/main.css [hunk 3]
```diff
@@ -97,34 +103,11 @@ h4 {
...
-  padding-left: 10px;
-  padding-right: 10px;
-}
-
-.navbar-right {
-  padding-right: 5px;
-}
-
 pre {
     white-space: pre-wrap;       /* CSS 3 */
```
Comment (mowchan) [inline css/main.css]: You should decrease the indentation here to match the rest of your styles. 

Also, you don't need to keep the comments.

File: css/main.css [hunk 4]
```diff
@@ -1,3 +1,8 @@
+html {
+  position: relative;
+  min-height: 100%;
+}
+
 body {
   background-image: url('https://ununsplash.imgix.net/photo-1415045384817-2f9cf7f2ed79?q=75&fm=jpg&s=a0d8ebf44a449c7855ca4128e914a7c5');
     background-repeat: no-repeat;
```
Comment (mowchan) [inline css/main.css]: Indentation

File: css/main.css [hunk 5]
```diff
@@ -7,6 +12,7 @@ body {
     -o-background-size: cover;
     background-size: cover;
```
Comment (mowchan) [inline css/main.css]: I'm not a huge fan of the way you indented these styles on the background-image. Personally, I would leave them all at the same level of indentation.

**Comment by TomWoodward** [comment]
looks :cake: to me @mowchan ?

**Comment by mowchan** [comment]
:cake:

**Pull request merged**